app/static/src/app/store/model/mutations.ts

```diff
@@ -1,39 +1,25 @@
 import { MutationTree } from "vuex";
-import { ModelState, RequiredModelAction } from "./state";
+import { ModelState } from "./state";
 import {
     Odin,
     OdinModelResponse,
-    OdinRunner,
-    OdinSolution,
-    WodinError
+    OdinRunner
 } from "../../types/responseTypes";
 import { evaluateScript, getCodeErrorFromResponse } from "../../utils";
-import { Dict } from "../../types/utilTypes";
 import { Palette } from "../../palette";
 
 export enum ModelMutation {
     SetOdinRunner = "SetOdinRunner",
     SetOdinResponse = "SetOdinResponse",
     SetOdin = "SetOdin",
-    SetOdinSolution = "SetOdinSolution",
-    SetRequiredAction = "SetRequiredAction",
-    SetParameterValues = "SetParameterValues",
-    UpdateParameterValues = "UpdateParameterValues",
-    SetPaletteModel = "SetPaletteModel",
-    SetEndTime = "SetEndTime",
-    SetOdinRunnerError = "SetOdinRunnerError"
+    SetCompileRequired = "SetCompileRequired",
+    SetPaletteModel = "SetPaletteModel"
 }
 
-const runRequired = (state: ModelState) => {
-    if (state.requiredAction !== RequiredModelAction.Compile) {
-        state.requiredAction = RequiredModelAction.Run;
-    }
-};
-
 export const mutations: MutationTree<ModelState> = {
     [ModelMutation.SetOdinRunner](state: ModelState, payload: string) {
         state.odinRunner = evaluateScript<OdinRunner>(payload);
-        state.odinRunnerError = null;
+        // state.odinRunnerError = null; // TODO: dispatch to clear this?
```

Comment:
To be honest, I think this is quite an unlikely eventuality - that you would have a run error and then would set the runner. I suppose if there'd been a problem accessing the api initially? It could clear the error from the action which sets this, but it's probably not all that important.